package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "h5o-chrome",
-  "version": "0.8.13",
+  "version": "0.8.14",
```

Comment:
Can you please revert this version update? I bump and tag before the release, to make it happen automatically.